docs/build.md

```diff
@@ -97,25 +97,6 @@ On macOS it is necessary to build and manually install the respective native lib
 ./build.sh CompileNative
 ```
 
-# Building Avalonia into a local NuGet cache
```

Comment:
Can you keep a crosslink to the "Building to the Local Cache" page? Local cache builds are extremely useful for small changes that any contribution might want to test locally in another project. It's not obvious that "nuget.md" is useful for them, since they might not need to create any actual releases.

Reply:
Done.